core/admin/mailu/internal/views/postfix.py

```diff
@@ -158,11 +159,13 @@ def postfix_sender_rate(sender):
 def postfix_sender_access(sender):
     """ Simply reject any sender that pretends to be from a local domain
     """
-    if not is_void_address(sender):
-        localpart, domain_name = models.Email.resolve_domain(sender)
-        return flask.jsonify("REJECT") if models.Domain.query.get(domain_name) else flask.abort(404)
-    else:
-        return flask.abort(404)
+    try:
+        if not is_void_address(sender):
+            localpart, domain_name = models.Email.resolve_domain(sender)
+            return flask.jsonify("REJECT") if models.Domain.query.get(domain_name) else flask.abort(404)
+    except sqlalchemy.exc.StatementError:
+        pass
+    return flask.abort(404)
```

Comment:
Hmm. I don't think this was correct in the first place.
The function is_void address is really just checking if there's an '@' in sender.
But it strips '<' and '>' - do we need to do this here, too?

Maybe 'is_void address' can be removed and 'postfix_sender_access' should be this

```
if '@' in email:
    if email.startswith('<') and email.endswith('>'):
        email = email[1:-1]
    try:
        localpart, domain_name = models.Email.resolve_domain(sender)
        if models.Domain.query.get(domain_name):
            return flask.jsonify("REJECT")
    except sqlalchemy.exc.StatementError:
        pass
return flask.abort(404)
```

Reply:
cd3eee4c